Source/WebCore/page/DragController.cpp

```diff
@@ -1263,13 +1260,14 @@ bool DragController::startDrag(LocalFrame& src, const DragState& state, OptionSe
 #endif
 
 #if ENABLE(INPUT_TYPE_COLOR)
-    bool isColorControl = is<HTMLInputElement>(state.source) && downcast<HTMLInputElement>(*state.source).isColorControl();
-    if (isColorControl && m_dragSourceAction.contains(DragSourceAction::Color)) {
-        auto& input = downcast<HTMLInputElement>(*state.source);
-        auto color = input.valueAsColor();
+    // FIXME: Should this use isEnabledColorInput instead?
```

Comment:
Not a fan of adding questions in the code like this, unless something is obviously wrong. Here, it's not obvious to me there is an issue.